tests/foreman/ui/test_dashboard.py

```diff
@@ -131,14 +131,15 @@ def test_positive_host_configuration_chart(session):
         session.organization.select(org_name=org.name)
         session.location.select(loc_name=loc.name)
         dashboard_values = session.dashboard.read('HostConfigurationChart')
-        assert dashboard_values['chart']['No report'] == '100%'
+        assert dashboard_values['chart'][''] == '100%'
```

Comment:
this is weird we are not having the key with associated. 
`assert '100%' in test['chart'].values()`  can't we do like this to avoid the dependency on the key? what're your thoughts?

Reply:
Agreed with Omkar, this is puzzling. But not blocking.